zerver/tests/test_slack_importer.py

```diff
@@ -233,12 +244,38 @@ def test_users_to_zerver_userprofile(self, mock_get_data_file: mock.Mock) -> Non
                       "team_domain": "foreignteam",
                       "profile": {"image_32": "https://secure.gravatar.com/avatar/random6.png",
                                   "avatar_hash": "hash", "first_name": "Matt", "last_name": "Perry",
-                                  "real_name": "Matt Perry", "display_name": "matt.perry", "team": "T6LARQE2Z"}}]
+                                  "real_name": "Matt Perry", "display_name": "matt.perry", "team": "T6LARQE2Z"}},
+                     {"id": "U8VAHEVUY",
+                      "team_id": "T5YFFM2QY",
+                      "name": "showell30",
+                      "real_name": "Steve Howell",
```

Comment:
@Tlazypanda I think you should use fake names and email addres. Not the real ones :)